tools/spec-gen-sdk/src/utils/readme.ts

```diff
@@ -27,27 +26,19 @@ export interface RepositoryConfiguration {
   after_scripts: string[];
 }
 
-function findSwaggerToSDKYamlBlocks(parsedMarkdownNode: commonmark.Node | undefined | null): commonmark.Node[] {
-  const result: commonmark.Node[] = [];
-  if (parsedMarkdownNode) {
-    const nodesToVisit: commonmark.Node[] = [parsedMarkdownNode];
-    while (nodesToVisit.length > 0) {
-      const node: commonmark.Node = nodesToVisit.shift()!;
-
-      if (node.firstChild) {
-        nodesToVisit.push(node.firstChild);
-      }
-      if (node.next) {
-        nodesToVisit.push(node.next);
-      }
-
-      if (node.type === "code_block" && node.info && node.info.toLowerCase().indexOf("$(swagger-to-sdk)") !== -1) {
-        result.push(node);
-      }
-    }
+export function findSwaggerToSDKBlocks(parseMarkdown: string):{ info: string; content: string;}[] {
```

Comment:
This function returns all the yaml block instead of filter out the `swagger-to-sdk` section. If this is what you intended, change the function name accordingly.